third_party/tutorial.proto

```diff
@@ -59,14 +59,20 @@ message Link {
 
 message InlineContent {
   oneof content {
-    string text = 1;
-    string strong_text = 2;
-    string emphasized_text = 3;
-    string inline_code_text = 4;
+    StylizedText text = 6;
     Link link = 5;
   }
+
+  reserved 1, 2, 3, 4;
+  reserved "strong_text", "emphasized_text", "inline_code_text";
```

Comment:
I'm not sure if re-using field names but changing the index works (e.g. re-using "text" but changing the index from 1 to 6). Cassie?

Reply:
I'm not sure either. Compiles, but I don't know if that makes any difference. @cassierecher